index.js

```diff
@@ -31,10 +31,16 @@ function setNodeSelector(podConfig, nodeSelectors) {
         return;
     }
 
+    const selectors = Object.keys(nodeSelectors);
+
+    if (!selectors.length) {
```

Comment:
Can we combine with the previous `if`?  I think it might look cleaner:
```
if (!nodeSelectors || typeof nodeSelectors !== 'object' || !Object.keys(nodeSelectors).length) {
     return;
}
```

Reply:
I wanted to avoid doing `Object.keys(nodeSelectors)` twice.